nva-file-model/src/main/java/no/unit/nva/file/model/File.java

```diff
@@ -12,6 +12,9 @@
 
 import static java.util.Objects.isNull;
 
+/**
+ * An object that represents the description of a file.
```

Comment:
Does this mean it contains some file metadata related to NVA or any other service?